linuxdeploy-plugin-gtk.sh

```diff
@@ -128,6 +146,7 @@ else
     echo "$0: pkg-config/pkgconf not found in PATH, aborting"
     exit 1
 fi
+LIBRARY_PATH="$(search_library_path)"
```

Comment:
```suggestion
LD_GTK_LIBRARY_PATH="${LD_GTK_LIBRARY_PATH:-$(search_library_path)}"
```

For the record, I chose a more explicit name to distinguish the variable from others set on the host, given the fact it now allows for user input. If you agree to the change, you'd need to search-and-replace the other places.

Reply:
Done in 09e234d26ed8bd88b33597a77420b60a70837945.